backup-and-restore.md

```diff
@@ -0,0 +1,71 @@
+Backup and restoration of application data can be achieved easily with the `aam`  (Anbox Application Manager) utility helper installed in the image. The `aam` can bundle any necessary application data together into a tarball file or uncompress the tarball file to a particular application folder according to the specified package name.
+
+## Back up application data
+
+See the following script for an example for backing up your data: 
+
+```bash
+ #!/bin/sh -ex
+aam backup com.canonical.candy
+TARBALL_FILE=$(basename $(find ./ -name *.tar.bz2))
+ # Upload the tarball to public or private cloud storage service
+curl -i -X POST --data-binary @"${TARBALL_FILE}" <cloud_storage_upload_url>
+```
+Running this script in an [addon backup hook](https://discourse.ubuntu.com/t/managing-addons/17759#heading--backup-and-restore) will back up the user data of a particular application with `aam` and upload the resulting tarball file to the cloud storage service when a container is stopped.
+
+If [`boot-package`](https://discourse.ubuntu.com/t/managing-applications/17760#heading--create-applications) can be retrieved from the apk file or is specified in the application manifest file, you can also back up the boot application data simply with the flag `--boot-package`.
+
+```bash
+$ aam backup --boot-package
```

Comment:
Are we removing leading `$` here or are you going to do a separate PR for this ?

Reply:
I'm planning to do that as a global change, but since this is a new file, it makes sense to fix it right away instead of adding it wrong first. Will update.